docs/admin/auth/methods.rst

```diff
@@ -134,18 +134,13 @@ Token must contain the following claims:
 
 The token's ``iss``, ``username`` and ``aud`` values must match the
 corresponding values within CrateDB.
-CrateDB uses either the values defined on a user level via ``CREATE USER`` or if
-absent the global configuration.
+
+CrateDB uses :ref:`jwt_defaults` if they are defined or values defined on a
+user level via ``CREATE USER`` if defaults are not defined.
```

Comment:
They aren't really defaults if its the only thing that will be used?

Maybe "global jwt settings"? What would @proddata call it?

Reply:
```suggestion
CrateDB uses the cluster-wide :ref:`jwt_defaults` if they are configured. If no cluster defaults are defined, it will use the values specified at the individual user level via the ``CREATE USER`` statement.
```